bot/exts/utils/snekbox/_cog.py

```diff
@@ -319,65 +274,123 @@ async def format_output(self, output: str) -> Tuple[str, Optional[str]]:
 
         return output, paste_link
 
+    def get_extensions_whitelist(self) -> set[str]:
+        """Return a set of whitelisted file extensions."""
+        return set(self.bot.filter_list_cache['FILE_FORMAT.True'].keys()) | TXT_LIKE_FILES
```

Comment:
Considering we don't currently allow users to send the extensions in `TXT_LIKE_FILES` I'm not sure it should be here.